apps/namadillo/src/atoms/notifications/functions.ts

```diff
@@ -1,13 +1,18 @@
 import { TxProps } from "@namada/types";
+import { EncodedTxData } from "lib/query";
 
 export const notificationIdSeparator = ";";
 
 export const createNotificationId = (
-  data?: TxProps | TxProps[] | string
+  data?: TxProps | TxProps[] | string | EncodedTxData<TxProps>
 ): string => {
   if (!data) return Date.now().toString();
   if (typeof data === "string") return data;
-
+  if ((data as EncodedTxData<TxProps>).type === "buildIbcTransfer") {
```

Comment:
Wouldn't be better to check first if "type" exists on data? I'm afraid this would throw an error trying to access `type` from an array

Reply:
Good call, I updated that.